docs/configuration/05-autocommands.md

```diff
@@ -9,3 +9,10 @@ lvim.autocommands.custom_groups = {
   { "BufWinEnter", "*.lua", "setlocal ts=8 sw=8" },
 }
 ```
+Using "*" will make the autocommand run in any file. 
+Autocommands runs console commands, in order to use vim commands, put the mode in front of the command
+``` lua
+lvim.autocommands.custom_groups = {
+  {"InsertEnter", "*", ":normal zz"},
```

Comment:
It may be clearer to remove the explanatory text before the code sample.  Instead, just add a comment explaining what it does.
``` lua
lvim.autocommands.custom_groups = {
   -- On entering a lua file, set the tab spacing and shift width to 8
   { "BufWinEnter", "*.lua", "setlocal ts=8 sw=8" },

  -- On entering insert mode in any file, scroll the window so the cursor line is centered
  {"InsertEnter", "*", ":normal zz"},
  }
  ```

Reply:
yes you're right, I changed it now